charts/neon-proxy/values.yaml

```diff
@@ -69,11 +71,11 @@ podLabels: {}
   # neon_service: proxy
   # neon_env: staging
   # neon_region: us-west-1
-  # neon_region_slug: virginia
+# neon_region_slug: virginia
```

Comment:
Are these indentations appropriate?

Reply:
seems like `docker run --rm --volume "$(pwd):/helm-docs" -u "$(id -u)" jnorwood/helm-docs:v1.9.1` does that.

Reply:
or not. Not sure.